onyo/tests/demo/reference_git_log.txt

```diff
@@ -131,37 +131,37 @@ Date:   Sun Jan 1 00:00:00 2023 +0100
     - warehouse/laptop_apple_macbook.oiw629 -> accounting/Bingo Bob/laptop_apple_macbook.oiw629
     - warehouse/monitor_dell_PH123.86JZho -> accounting/Bingo Bob/monitor_dell_PH123.86JZho
 
-commit 1890e9029d8b7560b485ae83d1ed41b136704fc0
+commit 145755d76ab2f5ea2b0d0d97d03ae562293fefd7
 Author: Yoko Onyo <yoko@onyo.org>
 Date:   Sun Jan 1 00:00:00 2023 +0100
 
-    new [1]: warehouse/headphones_apple_airpods.uzl8e1
+    new [1]: None
```

Comment:
Looks like a bug snuck in.

Is this known and will be fixed in the upcoming PRs?

Reply:
Good catch, thx! Missed that one. Need to fix.